src/data/int/parity.lean

```diff
@@ -195,6 +192,19 @@ begin
   simp with parity_simps
 end
 
+@[simp, norm_cast] theorem even_coe_nat (n : ℕ) : even (n : ℤ) ↔ even n :=
+by rw [even_iff, nat.even_iff, ←int.coe_nat_zero, ←int.coe_nat_one, ←coe_nat_bit0, ←coe_nat_mod,
+  coe_nat_inj']
+
+@[simp, norm_cast] lemma odd_coe_nat {n : ℕ} : odd (n : ℤ) ↔ odd n :=
+by rw [odd_iff, nat.odd_iff, ←int.coe_nat_one, ←coe_nat_bit0, ←coe_nat_mod, coe_nat_inj']
+
+@[simp] lemma nat_abs_even {n : ℤ} : even n.nat_abs ↔ even n :=
+coe_nat_dvd_left.symm
+
+@[simp] lemma nat_abs_odd {n : ℤ} : odd n.nat_abs ↔ odd n :=
+by rw [odd_iff_not_even, nat.odd_iff_not_even, nat_abs_even]
+
 -- Here are examples of how `parity_simps` can be used with `int`.
 
 example (m n : ℤ) (h : even m) : ¬ even (n + 3) ↔ even (m^2 + m + n) :=
```

Comment:
sorry, not that this is part of your PR but should we have `example`s? I've never seen one elsewhere in `mathlib`

Reply:
We have them occasionally, especially for verifying the tactics or type-class inference behave as expected. Strictly we should probably put them in files in `tests/`, but that's more effort, and easier to lose track of.